include/solvers/navier_stokes_base.h

```diff
@@ -443,9 +443,6 @@ class NavierStokesBase : public PhysicsSolver<VectorType>
   // Previous solutions vectors
   std::vector<VectorType> previous_solutions;
 
-  // Intermediary solution stages for SDIRK methods
-  std::vector<VectorType> solution_stages;
```

Comment:
I think all the multiphysics solvers have this parameter as well and needs to be deleted? Cahn Hilliard, VOF, heat transfer, tracer?